src/components/study/DownloadSection.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div class="download-section">
+    <h3 class="download-title">Available for download</h3>
+    <div class="links">
+      <div
+        v-for="(linkColumn, index) in displayedLinksColumns"
+        :key="index"
+      >
+        <div
+          v-for="({ urlKey, url }) in linkColumn"
+          :key="urlKey"
+          class="url-item"
+        >
+          <a class="url-title" :href="url">{{ buildTitle(urlKey) }}</a>
+          <span class="url-subtitle">{{ buildSubTitle(urlKey) }}</span>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup>
+import { computed } from 'vue';
+const props = defineProps({
+  studyUrls: Object
+});
+
+
+const displayedLinksColumns = computed(() => {
+  const firstColumn = ["briefPdf", "fullPdf"]
+    .map(urlKey => ({ urlKey, url: props.studyUrls[urlKey] }))
+    .filter(({ url }) => !! url);
+
+  const secondColumn = ["ecoXlsx", "socialXlsx", "acvXlsx"]
+    .map(urlKey => ({ urlKey, url: props.studyUrls[urlKey] }))
+    .filter(({ url }) => !! url);
+
+  return [firstColumn, secondColumn].filter(column => column.length !== 0)
+})
+
+function buildTitle(urlKey) {
+  switch (urlKey) {
+    case "briefPdf":
+      return "Study 6 page snapshot (pdf)";
+    case "fullPdf":
+      return "Study full report (pdf)";
+    case "ecoXlsx":
+      return "Economic data (xlsx)";
+    case "socialXlsx":
+      return "Social sustainability data (xlsx)";
+    case "acvXlsx":
+      return "Environmental data (xlsx)";
+    default:
+      console.error("Unrecognized url type");
+  }
+}
+function buildSubTitle(urlKey) {
+  switch (urlKey) {
+    case "ecoXlsx":
+      return "Contribution to economic growth and Inclusiveness";
+    case "briefPdf":
+    case "fullPdf":
+    case "socialXlsx":
+    case "acvXlsx":
+      return "";
+    default:
```

Comment:
Plutôt que de faire 2 fonctions avec 2 switch case, j'aurais plutôt fait un objet du genre 
`{
  "ecoXlsx" : {
    title : "Economic data (xlsx)",
    subtitle: "Contribution to economic growth and Inclusiveness",
  },
  briefPdf : {
    title : "Study 6 page snapshot (pdf)",
    subtitle: "",
  }
}`

Mais ta façon de faire marche aussi.

Reply:
@leonarf Bonne idée, j'ai regroupé titre/sous-titre et clé url même, de manière à rendre bien clair le display : [`1e470f5` (#101)](https://github.com/leonarf/VCA4D/pull/101/commits/1e470f59d5bf8cdf9ae6b73f0d5fa5c33c3f4eda)

Qu'en penses tu ?

Reply:
C'est très bien je trouve